docs/scenarios/parametric_contributing.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Parametric System-tests
+
+Note: a more in-depth overview of parametric system-tests can be found in [parametric.md](parametric.md).
+
+**MUST:** Acquaint yourself with [how system tests work](parametric.md#architecture-how-system-tests-work) before proceeding.
+
+## Use cases
+
+Let's figure out if your feature is a good candidate to be tested with parametric system-tests.
+
+System-tests in general are great for assuring uniform behavior between tracers. There are two types of system-tests, [end-to-end](/docs/README.md) and [parametric](/docs/scenarios/parametric.md).
+
+The "parametric" in parametric system-tests stands for parameters. The original purpose of parametric scenarios is when a behavior must be tested across several different values for one or more parameters, usually different tracer configurations with some examples being [environment variable configuration effects on api methods, sampling, propagation, configuration, telemetry](/tests/parametric).
+
+If your usage does not require different parameter values, then [end-to-end system-tests](/docs/README.md) should be used as they will achieve the same level of behavior uniformity verification and test the feature on real world use cases, catching more issues.
+
+System-tests are **not** for testing internal or niche tracer behavior. Unit tests are a better fit for that case.
+
+## Getting set up
+
+We usually add new system tests when validating a new feature. This feature might not yet be implemented across all dd-trace libraries. If at least one library already supports the feature, you can verify your test by running it against that library.
+
+To begin we need to make sure system-tests run with a tracer that has implemented the feature being tested (published or on a branch).
+Follow [Binaries Documentation](../execute/binaries.md) for your particular tracer language to set this up.
+
+[Verify that you can run some parametric tests with your custom tracer](parametric.md#running-the-tests). Make sure some pass (no need to run the whole suite, you can stop the tests from running with `ctrl+c`). If you have an issue, checkout the [debugging section](parametric.md#debugging) to troubleshoot.
+
+## Writing the tests
+
+Now that we're all setup with a working test suite and a tracer with the implemented feature, we can begin writing the new tests.
+
+**MUST:** If you haven't yet, please acquaint yourself with [how system tests work](parametric.md#architecture-how-system-tests-work) before proceeding and reference it throughout this section.
```

Comment:
It's purposeful to address other comments you made earlier about explaining more of how system-tests work. Most of that info should go in parametric.md, so rather than essentially re-explain it here I reiterate that implementers should reference it if confused.